kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/modem/quectel/ec25/QuectelEC25ModemFactory.java

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
```

Comment:
Please see the comments to the class similar to this one.

Reply:
ok